### DISCUSSION TITLE
Are wheel forces available?

### DISCUSSION THREAD

**Discussion by James721884** [carla-simulator/carla#4376] [Q&A]
I want to display the forces at each wheel. Like this; 

https://www.youtube.com/watch?v=x9EeGjy9uZc&t

Is this already being calculated somewhere? If so do you know which file to look at?

**Comment by James721884** [comment]
I'm stuggling to pull read basic vehicle properites, just tried;

`world.player.VehiclePhysicsControl.mass`

but I get an error, AttributeError: 'Vehicle' object has no attribute 'VehiclePhysicsControl'

**Reply by corkyw10** [reply]
Hi, you should be able to get the VehiclePhysicsControl object by calling [this method](https://carla.readthedocs.io/en/0.9.11/python_api/#carla.Vehicle.get_physics_control) on the vehicle instead of trying to access it directly through the vehicle itself.
